### PR TITLE
AC Test fix - LRAC-11314 | master

### PR DESCRIPTION
**Pull request by calvinctvo**
https://issues.liferay.com/browse/LRAC-11314 (Test fix ticket)
Note: Sometimes the banner does not show up until clicking on a different setting tabs. By adding another action (clicking on Usage) can help prevent the flakiness of this test.

### CODE REVIEW

**Comment by calvinctvo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Workspace.testcase [hunk 1]
```diff
@@ -321,6 +321,9 @@ definition {
 			ACUtils.closeModal();
 		}
 
+		task ("Click on different setting tabs") {
+			ACSettings.goToUsage();
+
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Workspace.testcase:326]: Hey @calvinctvo I think you are missing a closing bracket here.

**Review by manuferreira — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Workspace.testcase (hunk 1 above)
Comment (calvinctvo) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Workspace.testcase]: thank you @manuferreira

**Comment by calvinctvo** [comment]
ci:test:sf

**Pull request closed without merging**